charts/astronomer/values.yaml

```diff
@@ -23,7 +23,7 @@ images:
     pullPolicy: IfNotPresent
   houston:
     repository: astronomerinc/ap-houston-api
-    tag: master
+    tag: main
```

Comment:
Not clear to me why we're not using a versioned tag here?

Reply:
Looks like it was change recently so Adam could get something deployed. Similar things to what we've been doing on the dev cluster. This will have to change back to a version before the next release.